app/src/main/AndroidManifest.xml

```diff
@@ -51,6 +51,9 @@
             android:name=".module.organizer.detail.OrganizerDetailActivity"
             android:label="@string/title_activity_organizer_detail"
             android:theme="@style/AppTheme.Main.Light" />
+        <meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="INSERT_YOUR_API_KEY"/>
```

Comment:
@iamareebjamal Please suggest how to hide the API key.

Reply:
Load it through the environment variables